### PR TITLE
Add level

### PR DESCRIPTION
**Pull request by latypova-alina**

### CODE REVIEW

File: db/migrate/20161121105039_add_level_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddLevelToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :level, :int, default: 1
```
Comment (vladimirbazhanov) [inline db/migrate/20161121105039_add_level_to_users.rb]: :int should be :integer as I know?

Comment (latypova-alina) [inline db/migrate/20161121105039_add_level_to_users.rb]: It changes into :integer automatically and it works) Should I rewrite it then?

File: app/views/games/index.html.slim [hunk 2]
```diff
@@ -5,6 +5,10 @@
 
     h3 Статистика
 
+    ' Уровень:
+    =current_user.level
```
Comment (vladimirbazhanov) [inline app/views/games/index.html.slim]: add space

File: app/models/user.rb [hunk 3]
```diff
@@ -40,6 +40,13 @@ def self.from_omniauth(auth)
     end
   end
 
+  def level
+    return "Новичок" if score.in?(0..100)
```
Comment (vladimirbazhanov) [inline app/models/user.rb]: can you use here
```
case
  when
  when
  when
end
``` ?

**Pull request merged**